dlt/core/__init__.py

```diff
@@ -36,9 +36,6 @@ def get_version(loaded_lib):
 def get_api_specific_file(version):
     """Return specific version api filename"""
     version_tuple = [int(num) for num in version.split('.')]
-    if version_tuple[-1] != 0:
-        # The mirror version does not exist, try to truncate
-        version_tuple = version_tuple[:-1] + [0]
```

Comment:
Is it possible to leave the logic? I think it can find the specific version first. If not found, use the original logic

Reply:
Sure can